src/adapters/MeetingsJSONAdapter.js

```diff
@@ -384,6 +394,31 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
     return captureStream;
   }
 
+  /**
+   * Returns available media devices.
+   *
+   * @param {string} type  String specifying the device type.
+   * Could take one of the following values: 'videoinput'|'audioinput'|'audiooutput'.
```

Comment:
the pipe `|` syntax can be used instead of string:
```js
* @param {'videoinput'|'audioinput'|'audiooutput'} type  String etc.
```

if that does not work (it should), you can just use comma as a delimiter:

> Possible values: videoinput, audioinput, audiooutput.

Reply:
you can now remove the "Could take..." line

Reply:
please remove the "Could take" line